at_client_mobile/lib/src/at_client_auth.dart

```diff
@@ -96,14 +94,15 @@ class AtClientAuthenticator implements AtClientAuth {
           logger
               .finer('generating encryption key pair and self encryption key');
           var encryptionKeyPair = _keyChainManager.generateKeyPair();
-          await _keyChainManager.putValue(atSign, keychainEncryptionPrivateKey,
-              encryptionKeyPair.privateKey.toString());
+          var atSignItem = await _keyChainManager.readAtsign(name: atSign) ?? AtsignKey(atSign: atSign);
           var encryptionPubKey = encryptionKeyPair.publicKey.toString();
-          await _keyChainManager.putValue(
-              atSign, keychainEncryptionPublicKey, encryptionPubKey);
           var selfEncryptionKey = EncryptionUtil.generateAESKey();
-          await _keyChainManager.putValue(
-              atSign, keychainSelfEncryptionKey, selfEncryptionKey);
+          atSignItem = atSignItem.copyWith(
+            encryptionPrivateKey: encryptionKeyPair.privateKey.toString(),
+            encryptionPublicKey: encryptionPubKey,
```

Comment:
For consistency, please replace the above line with `encryptionPublicKey: encryptionKeyPair.publicKey.toString();` (and remove the now unused `encryptionPubKey` local variable)